teerex/sgx-verify/src/lib.rs

```diff
@@ -378,21 +378,17 @@ pub struct SgxVerifiedReport {
 	pub build_mode: SgxBuildMode,
 }
 
-type SignatureAlgorithms = &'static [&'static webpki::SignatureAlgorithm];
+type SignatureAlgorithms = &'static [&'static dyn webpki::types::SignatureVerificationAlgorithm];
 static SUPPORTED_SIG_ALGS: SignatureAlgorithms = &[
-	//&webpki::ECDSA_P256_SHA256,
-	//&webpki::ECDSA_P256_SHA384,
-	//&webpki::ECDSA_P384_SHA256,
-	//&webpki::ECDSA_P384_SHA384,
```

Comment:
This is something that I didn't notice before. Why do we comment the ECDSA signature algorithms, when we are verify signatures below with it. @brenzi I guess this is you who did it?

Reply:
Don't remember. I guess our patch was incomplete at the time